packages/connect-thegraph-template/subgraph/subgraph.template.yaml

```diff
@@ -1,17 +1,20 @@
+# Do not edit subgraph.yaml, this is a generated file.
+# Instead, edit subgraph.template.yaml and run `yarn manifest-<network>-[staging]`
 specVersion: 0.0.2
-description: {{description}}
-repository: {{repository}}
+description: A template for Aragon subgraphs
+repository: https://github.com/aragon/connect
 schema:
   file: ./schema.graphql
 dataSources:
-{{#DAOFactoryDataSources}}
-  {{> DAOFactory.template.yaml}}
-{{/DAOFactoryDataSources}}
-{{#KernelDataSources}}
-  {{> Kernel.template.yaml}}
-{{/KernelDataSources}}
+{{> Organizations.yaml}}
+{{> OrganizationFactories.yaml}}
+{{> OrganizationTemplates.yaml}}
+{{> LegacyOrganizationTemplates.yaml}}
+{{> TokenFactories.yaml}}
+{{> Tokens.yaml}}
 templates:
   {{> Kernel.template.yaml}}
+  {{> MiniMeToken.template.yaml}}
```

Comment:
Maybe include `Kernel.template.yaml` with ! to show that can be used as a template as well?

Reply:
Not sure what you mean. If what you mean is this:

```
templates:
  {{!> Kernel.template.yaml}}
  {{> MiniMeToken.template.yaml}}
```

That would disable the dynamic Kernal data source, which would break things.